docs/api/structures/permission-request.md

```diff
@@ -1,4 +1,5 @@
 # PermissionRequest Object
 
+* `requestingOrigin` string - The origin you should treat the request as coming from. Note this may be different to `requestingUrl` in the case of child frames, you should use `requestingOrigin` as the "safe" one to check.
```

Comment:
```suggestion
* `requestingOrigin` string - The origin you should treat the request as coming from. Note this may be different to `requestingUrl` in the case of child frames. You should use `requestingOrigin` as the "safe" one to check.
```

comma splice